hack/test-cmd.sh

```diff
@@ -188,18 +189,17 @@ atomic-enterprise start \
   --images="${USE_IMAGES}"
 
 # ensure that DisabledFeatures aren't written to config files
-! grep -i '\<disabledFeatures\>' \
-	"${MASTER_CONFIG_DIR}/master-config.yaml" \
-	"${BASETMPDIR}/atomic.local.config/master/master-config.yaml" \
-	"${NODE_CONFIG_DIR}/node-config.yaml"
+os::cmd::expect_success_and_not_text "cat ${MASTER_CONFIG_DIR}/master-config.yaml" '\<disabledFeatures\>'
```

Comment:
This failed:
```
FAILURE after 0.010s: hack/test-cmd.sh:192: executing 'cat /tmp/openshift-cmd/openshift.local.config/master/master-config.yaml' expecting success and not text '\<disabledFeatures\>': the output content test failed
Standard output from the command:
apiLevels:
- v1
apiVersion: v1
assetConfig:
  extensionDevelopment: false
  extensionScripts: null
  extensionStylesheets: null
  extensions: null
  loggingPublicURL: ""
  logoutURL: ""
  masterPublicURL: https://127.0.0.1:28443
  metricsPublicURL: ""
  publicURL: https://127.0.0.1:28443/console/
  servingInfo:
    bindAddress: 127.0.0.1:28443
    bindNetwork: tcp4
    certFile: master.server.crt
    clientCA: ""
    keyFile: master.server.key
    maxRequestsInFlight: 0
    namedCertificates: null
    requestTimeoutSeconds: 0
controllerLeaseTTL: 0
controllers: '*'
corsAllowedOrigins:
- 127.0.0.1
- 127.0.0.1:28443
- localhost
disabledFeatures: null
...
There was no error output from the command.
```

Why was this working before? Why were there angle brackets in the `grep` string?